src/commands/utility/changeStatus.js

```diff
@@ -0,0 +1,55 @@
+const { SlashCommandBuilder } = require("discord.js");
+
+const MAPPED_STATUS_COMMANDS = {
+  'pr-no-merge': '🚫',
+  'pr-sos': '🆘',
+  'pr-draft': '🚧',
+  'pr-reviewing': '👀',
+  'pr-request-changes': '🔁',
+  'pr-request-review': '❗',
```

Comment:
This one have extra spaces, maybe not needed

Reply:
Which one has the extra space?

Reply:
I think is only a visual effect as the character takes more space than the others